srcpkgs/marco/template

```diff
@@ -1,24 +1,20 @@
 # Template file for 'marco'
 pkgname=marco
-version=1.26.0
+version=1.26.1
```

Comment:
1.27.0 is available

Reply:
odd numbered versions are considered as development versions for mate and not official releases